app/gui/src/dashboard/components/AriaComponents/Button/Button.tsx

```diff
@@ -337,7 +337,7 @@ const ButtonContent = memo(function ButtonContent(props: ButtonContentProps) {
         styles={styles}
         hideLoader={hideLoader}
       />
-      <Text color="inherit" truncate="1" className={styles.text()}>
+      <Text weight="custom" color="inherit" truncate="1" className={styles.text()}>
```

Comment:
On the second though - why did we add the <Text /> component here?

Reply:
oh right. mainly so I could get ellipsis + hover tooltip when the timezone is too long:
![image](https://github.com/user-attachments/assets/98749967-f742-4547-b89a-3c4e202c67a5)

... granted, that should not be a button since it is not interactable, I am just using it as a hack to have an icon and text that are (or are supposed to be) pre-aligned

Reply:
any better suggestions?

Reply:
Buttons (actions) shouldn't be truncated, and use active verbs. What does this button do?

Reply:
Yeah, lets use just text element instead of button here.
>  i guess ideally we could refactor this out to a style-only component

Yeah, we can add this pattern as a `pattern` (https://panda-css.com/docs/concepts/patterns) using `tv` function.

Also we do the same here so we can base the receipt on this: https://github.com/enso-org/enso/blob/wip/sergeigarin/flat-folder-structure-follow-up/app/gui/src/dashboard/components/Breadcrumbs/BreadcrumbItem.tsx#L206